frontends/web/src/routes/device/bitbox02/reset.tsx

```diff
@@ -85,7 +85,7 @@ class Reset extends Component<Props, State> {
         <Dialog
           open={activeDialog}
           title={t('reset.title')}
-          onClose={this.abort}
+          onClose={() => this.setState({ activeDialog: false, understand: false })}
```

Comment:
this would require changing all components again. I'll look into a different approach